cloudformation/config-store.yaml

```diff
@@ -41,11 +42,22 @@ Resources:
               SSEAlgorithm: aws:kms
               KMSMasterKeyID: !GetAtt KmsKey.Arn
 
+  # Prices are confidential, so use fake data outside of the production environment
+  FakePricesS3Object:
+    Condition: IsNotProduction
+    Type: Custom::S3Object
+    Properties:
+      ServiceToken: !GetAtt CustomS3ObjectResourceFunction.Arn
+      S3Object:
+        Bucket: !Ref ConfigBucket
+        Key: rate_tables/prices.csv
+        Body: !TextInclude ./cloudformation/fake-prices.csv
+
   UkVatConfigS3Object:
     Type: Custom::S3Object
     Properties:
       ServiceToken: !GetAtt CustomS3ObjectResourceFunction.Arn
       S3Object:
         Bucket: !Ref ConfigBucket
         Key: uk-vat.json
```

Comment:
Request unrelated to this ticket: Consider putting the VAT stuff into its own directory as well, just for consistency.  E.g. vat/uk-vat.json